test/unit/lib/censorBirthV1.js

```diff
@@ -0,0 +1,284 @@
+    designation: "Registrar",
+    superintendentSignature: "Superintendent Signature",
+    superintendentDesignation: "Superintendent Designation",
+    subdistrict: "Subdistrict town",
+    district: "District city",
+    administrativeArea: "Adminshire"
+  },
+  informant1: {
+    forenames: "Informant Forename1",
+    surname: "Informant1 Surname",
+    address: "Informant1 Address",
+    qualification: "Confidential Informant",
+    signature: "Here is a signature",
+    signatureIsMark: false
+  },
+  informant2: {
+    forenames: "Informant Forename2",
+    surname: "Informant2 Surname",
+    address: "Informant2 Address",
+    qualification: "Not a Confidential Informant",
+    signature: "Here is a signature",
+    signatureIsMark: false
+  },
+  child: {
+    originalPrefix: "Master",
+    prefix: "Mr",
+    forenames: "Expensive",
+    originalForenames: "Inconvenient",
+    surname: "Mistake",
+    originalSuffix: "SUFFIX 1",
+    suffix: "SUFFIX 2",
+    dateOfBirth: "2020-09-23",
+    sex: "Male",
+    birthplace: "Bath"
+  },
+  mother: {
+    prefix: "Ms",
+    forenames: "MOTHER FORENAME",
+    surname: "MOTHER SURNAME",
+    suffix: "N/A",
+    birthplace: "Exeter",
+    occupation: "Teacher",
+    aliases: [],
+    address: "ADDRESS 1",
+    maidenSurname: "MAIDEN SURNAME",
+    marriageSurname: "MARRIAGE SURNAME"
+  },
+  father: {
+    prefix: "Mr",
+    forenames: "FATHER FORENAME",
+    surname: "FATHER SURNAME",
+    suffix: "ESQ",
+    birthplace: "BRISTOL",
+    occupation: "Cleaner",
+    aliases: [],
+    deceased: false
+  },
+  dateOfDeclaration: "2020-10-23",
+  dateOfStatutoryDeclarationOfParentage: "2020-10-23",
+  statutoryDeclarationOfParentage: "",
+  dateOfNameUpdate: "2020-10-24",
+  status: {
+    blocked: false,
+    marginalNote: "MARGINAL NOTE",
+    cancelled: false,
+    correction: "None",
+    nameUpdate: "None",
+    onAuthorityOfRegistrarGeneral: false,
+    potentiallyFictitious: false,
+    praOrCourtOrder: "None",
+    reregistration: "None"
+  },
+  previousRegistration: undefined,
+  nextRegistration: undefined
+}
+
+const redactedBirthDataNotBlocked = {
+  id: 12345,
+  date: "2020-09-23",
+  entryNumber: null,
+  registrar: {
+    signature: null,
+    designation: null,
+    superintendentSignature: null,
+    superintendentDesignation: null,
+    subdistrict: "Subdistrict town",
+    district: "District city",
+    administrativeArea: "Adminshire"
+  },
+  informant1: {
+    forenames: null,
+    surname: null,
+    address: null,
+    qualification: "Confidential Informant",
+    signature: null,
+    signatureIsMark: null
+  },
+  informant2: {
+    forenames: null,
+    surname: null,
+    address: null,
+    qualification: "Not a Confidential Informant",
+    signature: null,
+    signatureIsMark: null
+  },
+  child: {
+    originalPrefix: "Master",
+    prefix: "Mr",
+    forenames: "Expensive",
+    originalForenames: null,
+    surname: "Mistake",
+    originalSuffix: "SUFFIX 1",
```

Comment:
Sorry, `originalSuffix` and `originalPrefix` should get redacted as well.

Reply:
Updated